### ISSUE TITLE
Cleanup compatibility issue with ReSharper 9

### ISSUE THREAD

**Issue by codecadwallader** [codecadwallader/codemaid#43]
Reported by refrom under #31 and moved to a new issue.

Hi,
I'm affected by this error in every C# file in my solutions.
I got:

[CodeMaid Diagnostic 03:20:51 ] WindowEventListener.OnWindowChange raised for 'C:\Data\ex\ex.cs'
[CodeMaid Diagnostic 03:20:52 ] CleanupActiveCodeCommand.OnExecute invoked
[CodeMaid Diagnostic 03:20:52 ] CodeCleanupManager.Cleanup started for 'C:\Data\ex\ex.cs'
[CodeMaid Diagnostic 03:20:52 ] TextEditorEventListener.OnLineChanged raised for 'C:\Data\ex\ex.cs'
[CodeMaid Handled Exception 03:20:52 ] Stopped cleaning 'ex.cs': System.Runtime.InteropServices.COMException (0x80004005): Error HRESULT E_FAIL has been returned from a call to a COM component.
at EnvDTE80.DTE2.ExecuteCommand(String CommandName, String CommandArgs)
at SteveCadwallader.CodeMaid.Logic.Cleaning.UsingStatementCleanupLogic.RemoveUnusedUsingStatements(TextDocument textDocument, Boolean isAutoSave)
at SteveCadwallader.CodeMaid.Logic.Cleaning.CodeCleanupManager.RunCodeCleanupCSharp(Document document, Boolean isAutoSave)
at SteveCadwallader.CodeMaid.Logic.Cleaning.CodeCleanupManager.<>c__DisplayClass3.b__1()
at SteveCadwallader.CodeMaid.Helpers.UndoTransactionHelper.Run(Func1 undoConditions, Action tryAction, Action1 catchAction)
[CodeMaid Diagnostic 03:20:52 ] TextEditorEventListener.OnLineChanged raised for 'C:\Data\ex\ex.cs'
[CodeMaid Diagnostic 03:21:19 ] ConfigurationCommand.OnExecute invoked

When I use "clean file": Ctrl+Space, M combination.
I use Visual Studio 2013, problem started after installing R#.
I also figured out that the problem starts only after running R# cleaning on file, because if I clean using CodeMaid just after opening file it works, but if I use R# on that file then it got an error from CodeMaid.
I works with default setting of CodeMaid and R#.
Cleaning all open file and all solution still works.
CodeMaid version 0.7.4 
R# version 8 and 9.

Thanks for help.
